### ISSUE TITLE
DOATToUnitVentilator node connection errors

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6273]
Issue overview
--------------
Not sure when this slipped in, but DOATToUnitVentilator is now throwing node connection errors.  Checked all of the DOATo* files and the others are OK.
```
   ** Severe  ** Node Connection Error, Node="ZONE1UNITVENTOAMIXEROUTLETNODE", The same node appears as a non-parent Inlet node more than once.
   **   ~~~   ** Reference Object=ZONEHVAC:UNITVENTILATOR-OA MIXER, Name=TU1
   **   ~~~   ** Reference Object=FAN:CONSTANTVOLUME, Name=ZONE1UNITVENTFAN
   ** Severe  ** Node Connection Error, Node="ZONE2UNITVENTOAMIXEROUTLETNODE", The same node appears as a non-parent Inlet node more than once.
   **   ~~~   ** Reference Object=FAN:CONSTANTVOLUME, Name=ZONE2UNITVENTFAN
   **   ~~~   ** Reference Object=ZONEHVAC:UNITVENTILATOR-OA MIXER, Name=TU2
   ** Severe  ** Node Connection Error, Node="ZONE3UNITVENTOAMIXEROUTLETNODE", The same node appears as a non-parent Inlet node more than once.
   **   ~~~   ** Reference Object=FAN:CONSTANTVOLUME, Name=ZONE3UNITVENTFAN
   **   ~~~   ** Reference Object=ZONEHVAC:UNITVENTILATOR-OA MIXER, Name=TU3
   ** Severe  ** Node Connection Error, Node="ZONE4UNITVENTAIRINLETNODE", The same node appears as a non-parent Inlet node more than once.
   **   ~~~   ** Reference Object=FAN:CONSTANTVOLUME, Name=ZONE4UNITVENTFAN
   **   ~~~   ** Reference Object=ZONEHVAC:UNITVENTILATOR-OA MIXER, Name=TU4
   ** Severe  ** Node Connection Error, Node="ZONE5UNITVENTAIRINLETNODE", The same node appears as a non-parent Inlet node more than once.
   **   ~~~   ** Reference Object=FAN:CONSTANTVOLUME, Name=ZONE5UNITVENTFAN
   **   ~~~   ** Reference Object=ZONEHVAC:UNITVENTILATOR-OA MIXER, Name=TU5
   ************* There were 5 node connection errors noted.
```

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus Version 8.8.0-ddaa5689de

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file testfiles\DOATToUnitVentilator.idf
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
The only commit since v8.7 that touched UnitVentilators.cc is #6248 but I can't see the diffs from that anymore 'cause the pages have been deleted already <sigh>.  Will try a build just before that merge and see if that's where it came in.

**Comment by mjwitte** [comment]
Found the problem (moved GetATMixer *after* a block that's checking if ATMixerExists), will fix along with other changes to multiple airloop stuff.

**Comment by mjwitte** [comment]
Closed via #6272
